### PR TITLE
Remove option method

### PR DESCRIPTION
**Pull request by sculove**
- remove option method

### CODE REVIEW

File: test/unit/component.spec.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import Component from "../../src/Component";
 class TestClass extends Component {
 	constructor(option) {
 		super();
-		this.option(option);
+		this.option = option;
```
Comment (mixed) [inline test/unit/component.spec.js]: GOOD. 👍 
Could you make testcase for `option` property?

Comment (mixed) [inline test/unit/component.spec.js]: Ah.. Currently, Did Testcase have breaking?

Comment (sculove) [inline test/unit/component.spec.js]: @mixed test is passed.

Comment (sculove) [inline test/unit/component.spec.js]: I should change `option` to options.

**Comment by sculove** [comment]
I missed code deletion for options. now I handle it!

**Pull request merged**